### PR TITLE
new user token

### PR DESCRIPTION
**Pull request by RDWimmers**
Update the workflow for getting the short-lived access token.

NUCLEI username and password will be deprecated. The user token to generate the new short-lived access token is available on https://nuclei.cemsbv.io/#/personal-access-tokens

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,18 +19,17 @@ Please note that to use `GeoDataFrames` from the `geopandas` library `cems-nucle
...
 ```
-* NUCLEI_USER
-    - Your NUCLEI user name
-* NUCLEI_PASSWORD
-    - Your NUCLEI user password
+* NUCLEI_TOKEN
+    - Your NUCLEI user token
 ```
 
+You can obtain your `NUCLEI_TOKEN` on [NUCLEI](https://nuclei.cemsbv.io/#/).
```
[the remark this answers is not in the document]
Comment (RDWimmers) [reply README.md:26]: Done

File: nuclei/utils.py [hunk 2]
```diff
@@ -277,57 +273,3 @@ def message_to_python_types(
 
 def to_json(data: str) -> dict:
```
Comment (ritchie46) [inline nuclei/utils.py:274]: Do we still use this function? It seems redundant with `json.loads`.

**Review by ritchie46 — COMMENTED** [review]
LGTM.

File: nuclei/utils.py (hunk 2 above)
Comment (RDWimmers) [reply nuclei/utils.py]: You are right. Removed it.

**Pull request merged**